hubblestack/utils/decorators/jinja.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
```

Comment:
Need to remove usage of Jinja filter. We don't need this.

Reply:
Removed.